src/commands/root.command.ts

```diff
@@ -14,26 +14,46 @@ const example = chalk.green
 const helptext = `
 Example Usage
 
-  $ ${example('topos-zkevm-demo init')}
-    Initialize the Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
+  $ ${example('topos-zkevm-demo install')}
+    Install Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
+
+  $ ${example('topos-zkevm-demo start')}
+    Start the local-zkEVM infra to get your development chain up and running.
 
   $ ${example('topos-zkevm-demo execute')}
     Execute the Topos-zkEVM-Demo demo script to deploy contracts and execute transactions.
 
-  $ ${example('topos-zkevm-demo get-receipt')}
-    Get a transaction receipt from a transaction hash.
+  $ ${example(
+    'topos-zkevm-demo generate merkle-proof 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5'
+  )}
+    Generate a receipt merkle proof for the 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5 transaction.
```

Comment:
nit: 
```suggestion
    Generate a receipt merkle proof for the 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5 transaction hash.
```

additionally, I think it is more readable if we swap the value and the name? (same for below) but maybe that's just me

Reply:
(same for below) also was applicable to the `verify merkle-proof` piece, but I guess that's ok :)